packages/hash/api/src/model/index.ts

```diff
@@ -13,6 +13,18 @@ import Page from "./page.model";
 import Block from "./block.model";
 import Link from "./link.model";
 import Aggregation from "./aggregation.model";
+import DataType from "./ontology/dataType.model";
```

Comment:
Same question as above about camelCase for files. I appreciate the other model classes are using camelCase so it might be tempting to try and have some form of local consistency but I'm not sure that's worth us going ahead with something we don't want in the future.

Reply:
Yep, totally fair, addressed in c87b2141f5f5c65256d73783e312495453e66411